### DISCUSSION TITLE
Commits not updating GitHub stats after over 24 hours

### DISCUSSION THREAD

**Discussion by AProvince** [anuraghazra/github-readme-stats#2629] [Q&A] [answered]
<img width="911" alt="Screen Shot 2023-04-21 at 6 35 03 PM" src="https://user-images.githubusercontent.com/105252044/233752381-d88910d5-a180-44a8-b0fb-cca894817f1a.png">

<img width="412" alt="Screen Shot 2023-04-21 at 6 36 31 PM" src="https://user-images.githubusercontent.com/105252044/233752395-18ab299c-03a6-4760-97a9-3b11838d4680.png">

Hi, my apologies as I am new to GitHub. I am working on a Jekyll site using Alshedivat's template. For some reason, after making numerous commits over the course of the last two weeks, my GitHub profile stats still show 0 commits. Am I doing something wrong/misunderstanding?

**Comment by qwerty541** [comment, marked as answer]
Github does not count commits in forked repositories as contributions. This project uses the Github API under the hood, so here the commits made to the forked repository are also not taken into account.  You can read more about how Github count contributions there: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

**Reply by AProvince** [reply]
Thank you very much!

**Reply by qwerty541** [reply]
You are welcome! Mark this discussion as answered if you don't need help anymore.
